src/geo.c

```diff
@@ -211,7 +212,12 @@ int extractBoxOrReply(client *c, robj **argv, double *conversion,
  * the kilometer. */
 void addReplyDoubleDistance(client *c, double d) {
     char dbuf[128];
-    int dlen = snprintf(dbuf, sizeof(dbuf), "%.4f", d);
+
+    // instead of using snprintf "%.4f" we convert the double distance to long
```

Comment:
I think that comment now belongs in utils.c (geo.c doesn't convert to long, etc). 
Also, please use block comments (for some reason redis doesn't use line comments)